src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -24,111 +25,195 @@ def setUp(self):
         """Set up for test case."""
         super(OSAOfflineReintegration, self).setUp()
         self.dmg_command = self.get_dmg_command()
-        self.ior_apis = self.params.get("ior_api", '/run/ior/iorflags/*')
-        self.ior_test_sequence = self.params.get(
-            "ior_test_sequence", '/run/ior/iorflags/*')
-        self.ior_dfs_oclass = self.params.get(
-            "obj_class", '/run/ior/iorflags/*')
+        self.daos_command = DaosCommand(self.bin)
+        self.ior_test_sequence = self.params.get("ior_test_sequence",
+                                                 '/run/ior/iorflags/*')
+        self.test_oclass = self.params.get("oclass", '/run/test_obj_class/*')
+        self.loop_test_cnt = 1
```

Comment:
Can we define this via yaml.

Reply:
It is defined on YAML. Unfortunately, I don't want every test to run in a loop. It is going to take a long time. I am reading the loop test count for one of the weekly tests. For daily regressions, it is set to 1.